app/views/partnerships/index.html.erb

```diff
@@ -10,7 +10,7 @@
   <h1>Partner With <img src="<%= asset_path "rainbowdev.svg" %>" /> </h1>
   <h3>🚀 Reach</h3>
   <p>
-    DEV serves millions of unique visitors per month.  This highly-targeted developer audience is comprised of registered members (<%= User.count %>) and visitors from the open web — many of whom visit DEV as part of their daily routine.  You may be interested in the <a href="https://www.similarweb.com/website/dev.to">public analytics available on SimilarWeb</a>.
+    DEV serves millions of unique visitors per month.  This highly-targeted developer audience is comprised of registered members (<%= User.estimated_count %>) and visitors from the open web — many of whom visit DEV as part of their daily routine.  You may be interested in the <a href="https://www.similarweb.com/website/dev.to">public analytics available on SimilarWeb</a>.
```

Comment:
I don't think there's ever a good reason to display the exact count updated by the millisecond for things like the size of an entire table. This estimate also is accurate enough, see the algorithm up there